### ISSUE TITLE
Loss Increases after 10 epoches of training on coco

### ISSUE THREAD

**Issue by guanning03** [AILab-CVC/YOLO-World#441]
I tried the ordinary training code on ordinary coco dataset, but I found that the loss is increasing after 10 epoches. Has anyone met the same problem?
